contractcourt/chain_arbitrator.go

```diff
@@ -199,6 +199,9 @@ type ChainArbitratorConfig struct {
 
 	// HtlcNotifier is an interface that htlc events are sent to.
 	HtlcNotifier HtlcNotifier
+
+	// Budget is the configured budget for the arbitrator.
+	Budget *BudgetConfig
```

Comment:
Since this is not a state value but rather a read-only value, I'd like to advocate using a raw value here rather than a pointer.

Reply:
cool updated.